packages/design-tokens/src/js/themes/lightTheme/colors.ts

```diff
@@ -57,22 +57,22 @@ export const colors: ThemeColors = {
     inverse: brandColor.grey000,
   },
   primary: {
-    /** For primary semantic elements: interactive, active, selected (#0376C9) */
+    /** For primary semantic elements: interactive, active, selected... (#4459ff) */
```

Comment:
Not sure why the dots are here. Remove
```suggestion
    /** For primary semantic elements: interactive, active, selected (#4459ff) */
```

Reply:
Updated https://github.com/MetaMask/metamask-design-system/pull/474/commits/6bbb8da6bcf45b109eb113bf9a1e4367b6316a93